exec/csnex/Object.cs

```diff
@@ -179,11 +179,12 @@ public override bool subscript(Object index, ref Object r)
 
             if (Value.ContainsKey(s)) {
                 r = Value[s];
+                return true;
             }
             if (r == null) {
```

Comment:
I think this logic can be simplified, as I can no longer see a code path here where `r != null`.

Reply:
Fixed.  Removed the `r==null` logic.